rasa/cli/data.py

```diff
@@ -454,8 +465,8 @@ def _assert_nlu_pipeline_given(config: Dict, policy_names: List[Text]) -> None:
         for policy in [FallbackPolicy.__name__, TwoStageFallbackPolicy.__name__]
     ):
         rasa.shared.utils.cli.print_error_and_exit(
-            f"The model configuration has to include an NLU pipeline. This is required "
-            f"to migrate the fallback policies."
+            "The model configuration has to include an NLU pipeline. This is required "
+            "to migrate the fallback policies."
```

Comment:
```suggestion
            "in order to migrate the fallback policies."
```

Reply:
is this exit already tested?

Reply:
yes by `test_convert_config_with_missing_nlu_pipeline_config`